packages/frontend/src/view/ApiDefs/View/HealthCheckFailedAPI/HealthCheckFailedAPI.tsx

```diff
@@ -0,0 +1,13 @@
+import { EmptyContainer, EmptyText, Emoji, EmptyIcon } from '../../../../ui';
+import { HealthCheckFailedAPI as Props } from '../../../../types';
+
+export const HealthCheckFailedAPI: React.FC<Props> = ({ name }) => {
+  return (
+    <EmptyContainer>
+      <EmptyIcon />
+      <EmptyText>
+        Health check failed for {name} API <Emoji label="dizzy face"> 😵</Emoji>{' '}
```

Comment:
{name} API is not available at the moment. Try to refresh the page in a couple of seconds <Emoji label="dizzy face"> 😵</Emoji>